src/apiCalls.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+const fetchAllMovies = () => {
+  return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+  .then(res => res.json())
```

Comment:
One thing to check here is to make sure you're getting a 200-level status code back from the server here. That way you know you will likely be successful in parsing the data and getting it on the page. Check out `response.ok` as a way to check for this.